docs/Images/Tutorials/make_multicube_figures.py

```diff
@@ -273,8 +273,8 @@ def red_black_blue(mean_xyz):
                        flip_vector=[False, True, True])
     draw_basis_vectors(ax, [0.17, 2.17, 2.3], label_permutation=[2, 0, 1],
                        flip_vector=[False, True, True])
-    draw_basis_vectors(ax, [2.17, 2.17, 2.3], label_permutation=[0, 2, 1],
-                       flip_vector=[True, True, True])
+    draw_basis_vectors(ax, [2.17, 2.17, 2.3], label_permutation=[0, 1, 2],
```

Comment:
I would increase the z-position from 2.3 to 2.45, to (visually) separate the basis vectors from the overlapping cube.

Also you now use the default values of `label_permutation` and `flip_vector`, so you can just remove these args as in line 263.